### DISCUSSION TITLE
HUE upgrade from 4.3 to 4.10

### DISCUSSION THREAD

**Discussion by yaoyingyy** [cloudera/hue#3375] [Questions]
The HUE version I am currently using is 4.3.0, and I want to upgrade to 4.10. I have the following issues:
1. Is the metadata of the new and old versions compatible? If I upgrade from 4.3 to 4.10 or roll back from 4.10 to 4.3, will there be metadata incompatibility or bugs.
2. Are the configurations of the two versions compatible.

**Comment by Harshg999** [comment]
Hey @yaoyingyy, Hue 4.3 is years old. 
Yes, there can be breaking changes and stale/new configs.

**Reply by yaoyingyy** [reply]
Okay, thank you for your answer
